# HG changeset patch
# User jwe@segfault.lan
# Date 1201919224 18000
#      Fri Feb 01 21:27:04 2008 -0500
# Node ID 8386f7982831b821e92e3bd0088b3bb2651bd0d4
# Parent  5c2f5830179082f18658d5a9e68087938444581e
Added tag ss-2-9-1 for changeset 5c2f58301790

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -8,8 +8,9 @@ c1cdf2a32cdbf4e0d68aead4bfa597106e844989
 ce01dbd7e026c095d9e50beac529e8e955a303f8 ss-2-1-58
 1c65a8e44ef95fd4b2b11eb19071e3ecbbe77b76 ss-2-1-59
 1e6f653ef1e332fbaab48ac44149b0eb2d3bcfc8 ss-2-1-61
 1bf9abc0256be1800b553acdb30b099b4c2a3abb ss-2-1-62
 8386cf9811ee90243a965a3782b1b4f6cb107845 ss-2-1-63
 5a92c3177fc6c038bc93745f6ebc6a055e602b96 before-gnuplot-split
 e2ed74b9bfa052ce499b7f419e8d204123c49af0 after-gnuplot-split
 996a08a3eb068ce8002f54170fae755f9142d9a4 ss-2-9-0
+5c2f5830179082f18658d5a9e68087938444581e ss-2-9-1
